django/core/management/commands/populate_memberprofile_affiliations.py

```diff
@@ -0,0 +1,75 @@
+from django.core.management.base import BaseCommand
+from fuzzywuzzy import fuzz
+import re
+import requests
+import time
+import json
+import logging
+
+from core.models import MemberProfile
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = """Migrate data from MemberProfile.institution to MemberProfile.affiliations with
+    an attempt to add more data from ROR database"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--cached",
+            type=str
+        )
+
+    def handle(self, *args, **options):
+        # if options["cached"]:
+        #     with open(options["cached"]) as file:
+        #         data = json.load(file)
+
+        for mp in MemberProfile.objects.all().order_by('user_id'):
+            if mp.institution:
+                new_affil = {}
+
+                if mp.institution.name:
+                    new_affil["name"] = mp.institution.name
+                    best_match = self.lookup(mp.institution.name)
+
+                    if best_match:
+                        if (best_match["score"] >= 1.0 and fuzz.partial_ratio(
+                            best_match["organization"]["name"], mp.institution.name) >= 75):
```

Comment:
consider turning the magic constant 75 into an argparse argument with a default value of 75